cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -31,4 +31,7 @@ clusterStacks:
     deploy: false
 
 hostedConfig:
-  enabled: false
\ No newline at end of file
+  enabled: false
+
+templateStacks:
```

Comment:
I am kind of confused. [Comment here](https://github.com/crossplaneio/crossplane/pull/1214#discussion_r373614885) made sense to me since templating is one of the things that this mechanism will do but we call it template stack in other places. Should we converge on a naming for everywhere?

Reply:
Yeah, this is a good point. It's a bit confusing. The reason this part is using the word "template" is because "template stacks" has been the working Product Name for the functionality we're adding. Since this part is external facing, I used template stacks here too. I agree we should converge on a name that we use consistently.